tests/test_example.py

```diff
@@ -29,13 +30,19 @@
 
 def test_version(capsys):
     """Verify that version string sent to stdout, and agrees with the module."""
+    project_version = example.__version__
     with pytest.raises(SystemExit):
         with patch.object(sys, "argv", ["bogus", "--version"]):
             example.example.main()
     captured = capsys.readouterr()
     assert (
-        captured.out == f"{example.__version__}\n"
+        captured.out == f"{project_version}\n"
     ), "standard output by '--version' should agree with module.__version__"
+    travis_tag = os.getenv("TRAVIS_TAG")
+    if travis_tag:
+        assert (
+            travis_tag == project_version or travis_tag == f"v{project_version}"
```

Comment:
Why is the Travis tag allowed to be `v1.2.3` or `1.2.3`?  Is there a reason why we can't choose one format and apply it everywhere?

Reply:
I agree.  It's here because we didn't have a formal decision.  So which is it `v` or no `v`? 
less is more.  I think no `v`.

Reply:
Looking at respected projects: 

No `v`:
- https://github.com/python/black/tags
- https://github.com/pallets/flask/tags

Use `v`:
- https://github.com/python/cpython/tags
- HashiCorp (all)
- https://github.com/pfsense/pfsense/tags
- https://github.com/certbot/certbot/tags
- https://github.com/ansible/ansible/tags
- https://github.com/facebook/react/tags

So I think we should do this: 
- https://github.com/apple/swift/releases

After researching this I'm fine with the `v`.

Reply:
Yeah, I agree that the `v` is more of a standard than the no-`v`.